internal/hcs/process.go

```diff
@@ -190,22 +228,27 @@ func (process *Process) ResizeConsole(width, height uint16) error {
 	modifyRequestStr := string(modifyRequestb)
 
 	var resultp *uint16
+	completed := false
+	go syscallWatcher(process.logGetCommonFields(operation), &completed)
```

Comment:
Seems like there's a functional change in here. Should this be a separate review?

Reply:
Yes. I will submit these in another review. It looks like @jhowardmsft missed a few with his `syscallWatcher` so I will PR these up as bugs

Reply:
#388